### ISSUE TITLE
Ammo Skip not available on cylinder weapons, only drum

### ISSUE THREAD

**Issue by ProfessorOcecat** [chummer5a/chummer5a#4173]
The rules for the ammo skip modification state that it can be put on any gun with a cylinder or drum. It comes standard on the Lemat revolver (which it shows) but when attempting to add it to other revolvers it was not an option, even with no other modifications. However a weapon with a drum can receive it normally.

**To Reproduce**
Purchase a revolver (other than the Lemat)
Attempt to add ammo skip modification (its not an option)
Purchase a weapon with a drum mag
Attempt to add ammo skip (it is an option)

**Expected behavior**
Simply put: Ammo skip should be available for revolvers.

**Desktop (please complete the following information):**
Windows 10... not sure what version.

**Additional context**
During character creation

**Comment by chummer5a** [comment]
Revolvers don't have the necessary slot for the ammo skip mod, since it's an Under mod. Revolvers only have         Barrel, Top, Stock and Side mounts. I've added a customdata rule that will set the ammo skip to also allow consuming the Internal slot.
